### ISSUE TITLE
onlyLinks div disappears on small screens

### ISSUE THREAD

**Issue by rgshareholder** [vanila-io/mention-ghost-theme#2]
This key div disappears when site is viewed on an iphone/ipad in the vertical configuration.

**Comment by shtefcs** [comment]
Which one ? Can you share screenshot or something ?

**Comment by shtefcs** [comment]
@jelenajjo

**Comment by jelenajjo** [comment]
@rgshareholder Yes, I am aware of that and I will fix it in the next version. :) Thanks for bringing that up.
